test/image/mocks/bar_attrs_relative.json

```diff
@@ -9,7 +9,7 @@
             "type":"bar"
         }, {
             "width":[0.4,0.6,0.8,1],
-            "text":["Three",2,"inside text",0],
```

Comment:
Hmm, I think we need to figure out how to handle these cases. If I put values in where you put `""` - `"zero"` here and `"empty"` in the last trace - I get the following unfortunate situation:
<img width="337" alt="screen shot 2018-10-29 at 11 23 46 am" src="https://user-images.githubusercontent.com/2678795/47660497-80958180-db6d-11e8-8328-40834553633e.png">
I'm not sure why "zero" is showing up below where its bar would have stacked, seems like it should be above and it should push "4" inside the blue bar. "empty" also shows up below where its bar would be if that bar were positive... in an ideal world we might detect that its neighboring bars are negative - in this case only "outside text" to the right, but if it weren't the first bar, perhaps we would require the bar before to also be negative - we treat this as a "negative zero" and move the "empty" label down below the green -1? Anyway that's a pretty weird edge case, I wouldn't worry about it too much, if we could get it to work right with only a "positive zero" I'd be satisfied.

Reply:
I guess that the "zero" and " empty" text is showing up near the empty red bars is the new expected behavior. However, `textposition: 'auto'` is placing 'zero' below the empty bar and not pushing '4' inside. I'll look around and see if the blank-path solution solves this any better.